packages/sundials/meta.yaml

```diff
@@ -0,0 +1,27 @@
+package:
+  name: sundials
+  version: 7.1.1
+
+source:
+  url: https://github.com/LLNL/sundials/releases/download/v7.1.1/sundials-7.1.1.tar.gz
+
+  sha256: ea7d6edfb52448ddfdc1ec48f89a721fe6c0a259c10f8ef56f60fcded87a94bb
+
+requirements:
+  host:
+    - openblas
+    - suitesparse
+
+build:
+  type: shared_library
+  script: |
+    mkdir build
+    cd build
+
+    emcmake cmake .. -DBUILD_ARKCODE=OFF -DBUILD_CVODE=OFF -DEXAMPLES_ENABLE_C=OFF -DENABLE_KLU=ON -DEXAMPLES_INSTALL=OFF -DKLU_LIBRARY_DIR=${WASM_LIBRARY_DIR}/lib/ -DKLU_INCLUDE_DIR=${WASM_LIBRARY_DIR}/include
```

Comment:
```suggestion
    emcmake cmake .. -DBUILD_ARKODE=OFF -DBUILD_CVODE=OFF -DEXAMPLES_ENABLE_C=OFF -DENABLE_KLU=ON -DEXAMPLES_INSTALL=OFF -DKLU_LIBRARY_DIR=${WASM_LIBRARY_DIR}/lib/ -DKLU_INCLUDE_DIR=${WASM_LIBRARY_DIR}/include
```

Reply:
Should also probably disable `KINSOL` and `IDA`.

Reply:
Yes, if AMICI doesn't require them, please do